src/EventService.Models.Dto/CategoryInfo.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using LT.DigitalOffice.EventService.Models.Dto.Enums;
+
+namespace LT.DigitalOffice.EventService.Models.Dto
+{
+  public class CategoryInfo
+  {
+    public Guid Id { get; set; }
+    public string Name { get; set; }
+    public Color Color { get; set; }
+  }
+}
```

Comment:
Replace into new folder "Models"
It should look like Models.Dto -> Models ->CategoryInfo